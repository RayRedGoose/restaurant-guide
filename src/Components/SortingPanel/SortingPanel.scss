.sorting-panel {
  display: flex;
  justify-content: center;
  padding: 1rem;
  margin: 3rem 0;
  &>* {
    margin: 0 0.5rem;
  }
}
