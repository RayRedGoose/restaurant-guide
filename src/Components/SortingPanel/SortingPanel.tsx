import "./SortingPanel.scss";
import React from "react";
import Filter from "Containers/Filter/Filter";
import SearchForm from "Containers/SearchForm/SearchForm";
import CleanFiltersButton from "Containers/CleanFiltersButton/CleanFIltersButton";

const SortingPanel: React.FC = () => (
  <header className="sorting-panel">
    <SearchForm />
    <Filter type="states" />
    <Filter type="genres" />
    <Filter type="attires" />
    <CleanFiltersButton />
  </header>
);

export default SortingPanel;
