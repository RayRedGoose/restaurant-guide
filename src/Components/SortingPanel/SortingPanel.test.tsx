import React from 'react';
import { shallow } from 'enzyme';
import SortingPanel from './SortingPanel';

it('should match the snapshot', () => {
  const wrapper = shallow(<SortingPanel />);
  expect(wrapper).toMatchSnapshot();
});
