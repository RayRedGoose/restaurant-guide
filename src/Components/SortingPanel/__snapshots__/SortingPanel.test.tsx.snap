// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<header
  className="sorting-panel"
>
  <Filter
    type="states"
  />
  <Filter
    type="genres"
  />
  <Filter
    type="attires"
  />
  <SearchForm />
</header>
`;
